maxide.bmx

```diff
@@ -87,7 +87,7 @@ Const EOL$="~n"
 
 Const ABOUT$=..
 "{bcc_version}.~n~n"+..
-"Please visit www.blitzmax.org for all your BlitzMax related needs!"
+"Please visit https://blitzmax.org for all your BlitzMax related needs!"

```

Comment:
I'm surprised the about text isn't localised too :-)

Reply:
Can add a little commit to fix that ;-)

Reply:
:laughing: 
That line ... isn't even used in maxide.bmx ...

Reply:
Oh? Are you sure? We should just remove it if it's not used, because it's just taking up memory otherwise.